packages/components/src/components/post-popovercontainer/post-popovercontainer.tsx

```diff
@@ -69,13 +62,30 @@ export class PostPopovercontainer {
    */
   @Prop() readonly arrow?: boolean = false;
 
+  /**
+   * Enables a safespace through which the cursor can be moved without the popover being disabled
+   */
+  @Prop() readonly safeSpace?: 'triangle' | 'trapezoid';
```

Comment:
If you use [reflection](https://stenciljs.com/docs/properties#reflect-properties-values-to-attributes-reflect), you can use the prop as a selector in CSS and don't have to write a data-attribute for it, duplicating the setting.

```suggestion
  @Prop({ reflect: true }) readonly safeSpace?: 'triangle' | 'trapezoid';
```